tutorials/jmsActiveMQ/README.md

```diff
@@ -0,0 +1,49 @@
+## Sample jmsActiveMQ Read Application
+
+This application reads from a queue in `ActiveMQ`, creates one line per message and writes these lines to a file
+in the user specified directory in HDFS.
+
+Follow these steps to run this application:
+
+**Step 1**: Set up the ActiveMQ Broker and the Queue to read from
+
+Unless you already have the ActiveMQ broker installed, download, install and
+start it as per http://activemq.apache.org/version-5-getting-started.html.
+For this example, let us assume the broker is installed on a machine with the IP address
+192.168.128.142. You can access the Admin UI as 
```

Comment:
Minor point but would be good to eliminate trailing whitespace in a few lines in this file.